static/app/views/performance/transactionSummary/transactionSpans/spanSummary/spanSummaryCharts.tsx

```diff
@@ -0,0 +1,177 @@
+
+function SpanSummaryCharts() {
+  const organization = useOrganization();
+  const {spanSlug} = useParams();
+  const [spanOp, groupId] = spanSlug.split(':');
+
+  const location = useLocation();
+  const {transaction} = location.query;
+
+  const filters: SpanMetricsQueryFilters = {
+    'span.group': groupId,
+    'span.op': spanOp,
+    transaction: transaction as string,
+  };
+
+  const {
+    isLoading: isThroughputDataLoading,
+    data: throughputData,
+    error: throughputError,
+  } = useSpanMetricsSeries({
+    search: MutableSearch.fromQueryObject(filters),
+    yAxis: ['spm()'],
+    enabled: Boolean(groupId),
+    referrer: 'api.starfish.span-summary-page-charts',
+  });
+
+  const {
+    isLoading: isAvgDurationDataLoading,
+    data: avgDurationData,
+    error: avgDurationError,
+  } = useSpanMetricsSeries({
+    search: MutableSearch.fromQueryObject(filters),
+    yAxis: [`avg(${SpanMetricsField.SPAN_SELF_TIME})`],
+    enabled: Boolean(groupId),
+    referrer: 'api.starfish.span-summary-page-charts',
+  });
+
+  const eventView = EventView.fromNewQueryWithLocation(
+    {
+      yAxis: ['count()'],
+      name: 'Transaction Throughput',
+      query: MutableSearch.fromQueryObject({
+        transaction: transaction as string,
+      }).formatString(),
+      fields: [],
+      version: 2,
+    },
+    location
+  );
+
+  const {
+    isLoading: isTxnThroughputDataLoading,
+    data: txnThroughputData,
+    error: txnThroughputError,
+  } = useGenericDiscoverQuery<
+    {
+      data: any[];
+      meta: MetaType;
+    },
+    DiscoverQueryProps
+  >({
+    route: 'events-stats',
+    eventView,
+    location,
+    orgSlug: organization.slug,
+    getRequestPayload: () => ({
+      ...eventView.getEventsAPIPayload(location),
+      yAxis: eventView.yAxis,
+      topEvents: eventView.topEvents,
+      excludeOther: 0,
+      partial: 1,
+      orderby: undefined,
+      interval: eventView.interval,
+    }),
+    options: {
+      refetchOnWindowFocus: false,
+    },
+    referrer: 'api.starfish.span-summary-page-charts',
+  });
+
+  const transactionSeries: Series = {
+    seriesName: 'Throughput',
+    data:
+      txnThroughputData?.data.map(datum => ({
+        value: datum[1][0].count,
+        name: datum[0],
+      })) ?? [],
+  };
+
+  return (
+    <BlockContainer>
+      <Block>
+        <ChartPanel title={t('Average Duration')}>
+          <Chart
+            height={160}
+            data={[avgDurationData?.[`avg(${SpanMetricsField.SPAN_SELF_TIME})`]]}
+            loading={isAvgDurationDataLoading}
+            type={ChartType.LINE}
+            definedAxisTicks={4}
+            aggregateOutputFormat="duration"
+            stacked
+            error={avgDurationError}
+            chartColors={[AVG_COLOR]}
+          />
+        </ChartPanel>
+      </Block>
+
+      <Block>
+        <ChartPanel title={t('Span Throughput')}>
```

Comment:
![Screenshot 2024-05-03 at 2 27 03 PM](https://github.com/getsentry/sentry/assets/63818634/5f730dab-c708-491e-8cfc-fa893fe0d102)
![Screenshot 2024-05-03 at 2 26 57 PM](https://github.com/getsentry/sentry/assets/63818634/f108abd7-2eb9-4cd4-82eb-02bb4631a1f3)

nit: let's make the tooltips consistent? it's a little weird that we'd call one `spm` and the other `throughtput`

Reply:
changed the tooltip to say `tpm()` for the transaction throughput chart